### PR TITLE
feat: add imperative present tense requirement

### LINKED ISSUE #84: Use imperative present tense verb in example
While Conventional Commits itself does not specify verb voice and tense for the description, it's common to use imperative, present tense describing what the commit, if applied, does, rather than past tense describing what was done. So adopt this convention in the example, making the commit message example more relatable.

Before: 

    ### Commit message with scope
    ```
    feat(lang): added polish language
    ```

After:

    ### Commit message with scope
    ```
    feat(lang): add polish language
    ```

See #65 for an implementation.

#77 would go further and recommend imperative present tense, rather than just modeling it in this example commit message.

### PR DESCRIPTION
**Pull request by hbetts**
Add imperative present tense requirement.

Closes #84 #77

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -94,12 +94,13 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 5. A description MUST immediately follow the type/scope prefix.
 The description is a short description of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
 6. A longer commit body MAY be provided after the short description, providing additional contextual information about the code changes. The body MUST begin one blank line after the description.
-7. A footer MAY be provided one blank line after the body (or after the description if body is missing).
+7. The description and commit body MUST be written in [imperative](https://en.wikipedia.org/wiki/Imperative_mood) present tense.
```
Comment (damianopetrungaro) [inline index.md]: Do you think that it _MUST be writter_?

Comment (hbetts) [reply index.md]: Not sure what you mean.

Are you asking whether the imperative form should be a requirement of the convention?

Comment (damianopetrungaro) [reply index.md]: Do you think that we should use _MUST_? I'd prefer _SHOULD_?
I know Several people that are "against" this way of thinking, so I think _SHOULD_ it's better.

Comment (hbetts) [reply index.md]: @damianopetrungaro you are right. I did [say that I would _recommend_ the imperative form](https://github.com/conventional-commits/conventionalcommits.org/pull/65#issuecomment-412357437).

I have updated the pull request.

Comment (damianopetrungaro) [reply index.md]: Thank you for the effort @hbetts !

**Comment by damianopetrungaro** [comment]
@conventional-commits/committee let's approve this and merge this one 😄

**Comment by hbetts** [comment]
This should probably be versioned, though I don't know the process to do that.

**Comment by damianopetrungaro** [comment]
It's not automated, we should work on it.

**Comment by hbetts** [comment]
This has been updated to change the requirement to a recommendation and to place it in the FAQ.

**Comment by damianopetrungaro** [comment]
Thank you for the effort @hbetts !

I'll read more accurately tomorrow morning 😄

File: index.md [hunk 2]
```diff
@@ -156,6 +156,12 @@ In a worst case scenario, it's not the end of the world if a commit lands that d
 No! If you use a squash based workflow on Git lead maintainers can cleanup the commit messages as they're merged—adding no workload to casual committers.
 A common workflow for this is to have your git system automatically squash commits from a pull request and present a form for the lead maintainer to enter the proper git commit message for the merge.
 
+### What writing form should I use?
+
+We recommend writing a commit description and body using the [imperative](https://en.wikipedia.org/wiki/Imperative_mood) present tense writing form.
+
+There is significant examples of this writing form for commits [1](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)[2](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#subject)[3](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project)[4](https://medium.com/@danielfeelfine/commit-verbs-101-why-i-like-to-use-this-and-why-you-should-also-like-it-d3ed2689ef70)[5](https://chris.beams.io/posts/git-commit/)
```
Comment (damianopetrungaro) [inline index.md]: There `are` significant examples of this writing form for commits

**Comment by damianopetrungaro** [comment]
___
Off Topic:
@zeke  and @hbetts about the versioning we can skip it for the moment, this is a `beta` and until we won't upgrade the 1.0.0 version but we'll just bump-up the beta number at the end of the versioning `1.0.0-beta.2`.

But we need to find a strategy to release when we'll be out of the beta version (and we are pretty close).

I'll open an issue right now.

File: index.md (hunk 2 above)
Comment (hbetts) [reply index.md]: Fixed

**Pull request merged**